### PR TITLE
Add missing checksums in foss-2016b easyconfigs

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/ifort/ifort-2016.3.210-GCC-5.4.0-2.26.eb [hunk 1]
```diff
@@ -10,7 +10,10 @@ toolchain = {'name': 'dummy', 'version': ''}
 
 sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_fortran_update%(version_minor)s.tgz']
 
-checksums = ['70cf1ea91280e3e8ba4bc216bae63e4a']
+checksums = [
+    '2ae7d2b65c9c71e3192c072a1ae4c286950a4f81107608e7a18777b57f37efa0',  # parallel_studio_xe_2016_composer_edition_for_fortran_update3.tgz
```
Comment (boegel) [inline easybuild/easyconfigs/i/ifort/ifort-2016.3.210-GCC-5.4.0-2.26.eb]: @vanzod line too long, please move comment above checksum

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 33 out of 45 (45 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/d5acebdf969350ea33ddf2146cf2050d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 37 out of 45 (45 easyconfigs in this PR)
node2105.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2527b9264e31c714e0bbe59bc619f2ed for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 27 out of 45 (45 easyconfigs in this PR)
node2130.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5f2bb1ee7f90b050819f4076049e0c61 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 45 out of 45 (45 easyconfigs in this PR)
node2129.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d8037b5a72b247b60b0471cf8877ee87 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**